lib/src/appengine_internal.dart

```diff
@@ -257,19 +294,24 @@ Future<memcache.Memcache> _obtainMemcacheInstance() async {
       // once we go out of the current service scope and return the instance.
       await memcacheService.remove(testKey);
       ss.registerScopeExitCallback(rawMemcache.close);
+      logging.info('Connected to memcache at $host:$port');
       return memcacheService;
     }
-  } catch (error) {
-    // FIXME(kustermann): We should log the [error] message here!
+  } catch (e, stack) {
+    logging.warning([
+      'Could not connect to memcache instance at $host:$port',
+      e,
+      stack
+    ].where((i) => i != null).join('\n'));
```

Comment:
Please remove this complicated `[].where....` and just assume `e` and `stack` are non-null.

It can read as simple as:
```dart
logging.warning('Could not connect to memcache instance at $host:$port\n$e\n$stack');
```

Reply:
done